synchronizations/Synchronization.md

```diff
@@ -0,0 +1,223 @@
+## Tutorial
+![EMF Diagrams](images/EMFDiagrams.png)
+
+We generate the corresponding Ecore files from the two class diagrams shown above. One for the family model and one for the person model. 
+We then generate the NMF models and classes, configure our .csproj file and load our models into the Program.cs file analogous to 
+[Starting our first project](../models/FirstNmfProject.md)
+
+The first step is to create a file in which we define the synchronization logic of the two models. We name this file `Families2PersonsSynchronization.cs`.
+
+Let's think about what we need for our example. For this, let's look at the class diagram and draw the following conclusions. We need the FamilyRegister, the PersonRegister, and the Synchronization class where the synchronization logic between Family and Person is defined.
+```csharp
+class Program
+    {
+        private FamilyRegister familyRegister;
+        private PersonRegister personRegister;
+
+        private ModelRepository repository = new ModelRepository();
+
+        private Families2PersonsSynchronization synchronization = new Families2PersonsSynchronization();
+
+        static void Main(string[] args)
+        {
+        }
+    }
+```
+
+Next, we need to consider how the initialization should look. In our example, this will take place in the class constructor.
+
+```csharp
+public Program()
+        {
+            var familyRootModel = new Model();
+            var personRootModel = new Model();
+
+            familyRegister = new FamilyRegister();
+            personRegister = new PersonRegister();
+
+            synchronization.Synchronize(
+                synchronization.SynchronizationRule<Families2PersonsSynchronization.FamilyRegisterToPersonRegister>(),
+                ref familyRegister, // Familyregister is the left model
+                ref personRegister, // Personregister is the right model
+                SynchronizationDirection.LeftWins, // prioritizes the left synchronization of left model (without shouldCorrespond it has no impact)
+                ChangePropagationMode.TwoWay); // TwoWay means Bidirectional synchronization 
+
+            familyRootModel.RootElements.Add(familyRegister);
+            personRootModel.RootElements.Add(personRegister);
+
+            repository.Models.Add(new Uri("ttc:source"), familyRootModel);
```

Comment:
Using the Uris TTC:source and TTC:target is specific to the TTC. One could easily use other URIs